stonesoup/sensor/passive.py

```diff
@@ -33,13 +33,13 @@ class PassiveElevationBearing(Sensor):
             (and follow in format) the underlying \
             :class:`~.CartesianToElevationBearing` model")
 
-    def measure(self, ground_truth, noise=True, **kwargs):
+    def measure(self, ground_truths, noise=True, **kwargs):
```

Comment:
Is it worth type hinting these?
```suggestion
    def measure(self, ground_truths: Set[State], noise: Union[np.ndarray, bool] = True, **kwargs) -> Set[Detection]:
```
or (depending on whether the text or the annotation is correct (see comment below)
```suggestion
    def measure(self, ground_truths: Set[GroundTruthState], noise: Union[np.ndarray, bool] = True, **kwargs) -> Set[Detection]:
```

Of course, there are a lot of places to apply this change...

Reply:
Would the type hints be necessary if the argument types are documented?